README.md

```diff
@@ -387,7 +387,7 @@ overhead during runtime, so for production environments we should unwrap this
 using [yo-yoify][yo-yoify].
 
 Setting up browserify transforms can sometimes be a bit of hassle; to make this
-more convenient we recommend using [bankai][bankai] with `--optimize` to
+more convenient we recommend using [bankai][bankai] with `--watch=false` to
```

Comment:
hmm, this flag isn't supposed to exist; we should mention the `bankai build` command instead

Reply:
@goto-bus-stop You mean it should read `bankai build` instead of just `bankai`?

Reply:
@goto-bus-stop Or do you mean that `bankai build` will optimize without `--watch=false`? It's unclear to me.

Reply:
the latter!

Reply:
@goto-bus-stop Gotcha, will update.